src/pages/goal/GoalDetail.tsx

```diff
@@ -111,19 +130,61 @@ const GoalDetail = () => {
               </div>
 
               {/* (4) 기한 */}
-              {/* <PropertyItem defaultValue="기한" iconMap={{ 기한: dateIconMap }} /> */}
+              <div
+                onClick={(e) => {
+                  e.stopPropagation();
+                  openDropdown({ name: 'date' });
+                }}
+                className="flex w-full h-[3.2rem] px-[0.5rem] rounded-md items-center gap-[0.8rem] mb-[1.6rem] whitespace-nowrap hover:bg-gray-200 cursor-pointer"
+              >
+                {/* '기한' 속성 아이콘 */}
+                <img src={IcCalendar} alt="date" />
+                <div className="relative">
+                  {/* '기한' 항목명 - 날짜 설정하면 반영됨 */}
+                  <span className={`font-body-r text-gray-600`}>{getDisplayText()}</span>
+                  {/* 달력 드롭다운 오픈 */}
+                  {isOpen && content?.name === 'date' && (
+                    <CalendarDropdown
+                      selectedDate={selectedDate}
+                      onSelect={(date) => setSelectedDate(date)}
+                    />
+                  )}
+                </div>
+              </div>
 
               {/* (5) 이슈 */}
-              {/**
-               * @todo
-               * - 이슈 많아질 경우 '외 n개'로 축약되게 하기
-               */}
-              {/*
-                <PropertyItem
-                  defaultValue="이슈"
-                  iconMap={{ 이슈: issueIconMap }}
-                />
-              */}
+              <div
+                onClick={(e) => {
+                  e.stopPropagation();
+                  openDropdown({ name: '이슈' });
+                }}
+                className={`flex w-full h-[3.2rem] px-[0.5rem] rounded-md items-center gap-[0.8rem] mb-[1.6rem] whitespace-nowrap hover:bg-gray-200 cursor-pointer`}
+              >
+                {/* 속성 아이콘 */}
+                <img src={IcIssue} alt="이슈" />
+
+                {/* 속성 이름 */}
+                <div className="flex relative">
+                  <div className="flex items-center">
+                    {/* 속성 항목명 */}
+                    <div className="font-body-r text-gray-600">{option}</div>
+                  </div>
```

Comment:
```suggestion
                 <p className="font-body-r text-gray-600 max-w-[27.4rem] truncate">{option}</p>
```
P1: 속성 항목명을 감싸는 상위 div 태그는 불필요한 것 같아 없애주시면 좋을 것 같습니다!
또한, 글이 들어가기 때문에 div 태그보다는 p태그를 사용하는게 더 적절할 것 같습니다.

truncate을 지정하려면 최대 너비를 지정해야 하므로 ArrowDropdown과 같은 너비인 27.4rem을 지정해주시면 될 것 같습니다!

Reply:
반영했습니다